docs/MockFunctions.md

```diff
@@ -127,6 +127,52 @@ dependent component and configuring that, but the technique is the same. In
 these cases, try to avoid the temptation to implement logic inside of any
 function that's not directly being tested.
 
+## Mocking Modules
+
+Suppose we have a class that fetches users from our API. The class uses
+[axios](https://github.com/axios/axios) to call the API then returns the `data`
+attribute which contains all the users:
+
+```js
+// users.js
+import axios from 'axios';
+
+class Users {
+  static all() {
+    return axios.get('/users.json').then(resp => resp.data);
+  }
+}
+
+export default Users;
+```
+
+Now, in order to test this method without actually hitting the API (and thus
+creating slow and fragile tests), we can use the `jest.mock(...)` function to
+automatically mock the axios module.
+
+Once we mock the module we can spy on the `.get` method and provide a
```

Comment:
we don't spy, what about "Once we mock the module we can provide a `mockImplementation` for `.get` that returns..."?

Reply:
@SimenB I updated to be more clear and mentioned `mockReturnValue` 👍